### PR TITLE
Add and configure service worker

### PR DESCRIPTION
**Pull request by oskarrough**
I've had trouble with service workers before. They make development harder. But they also make the site quite a bit faster and able to render something when offline.

But ember-service-worker recently added a feature to handle caching http://ember-service-worker.com/documentation/configuration/#versioning. So every time we `yarn build`, a new filename is generated and caching shouldn't be an issue. Let's see?

### CODE REVIEW

**Comment by hugurp** [comment]
Neetos!

Feedback:
- "no internet" message disappear when dummy app is removed

Feature idea:
- when offline, "file input" to select a folder with the offline tracks (and would work exactly like r4)

**Comment by oskarrough** [comment]
Right about the offline message. It only happens in production. In development the offline message stays there, meaning the app doesn't render. Don't know why.

Can test with `ember serve -prod`. Gotta find a solution for that before merging.

File: app/components/app-root/component.js [hunk 1]
```diff
@@ -21,34 +22,43 @@ export default Component.extend({
 	isShowingModal: false,
 
 	didInsertElement() {
-		run.scheduleOnce('afterRender', () => {
-			this.removeDummyHTML();
-		});
+		var connectedRef = firebase.database().ref('.info/connected')
+
+		// Don't remove dummy HTML if offline.
```
Comment (hugurp) [inline app/components/app-root/component.js]: instead, maybe we should remove the dummy, and show a ember component?

**Comment by hugurp** [comment]
It's perfect, let's go with that and make it a component when ready for some offline support.

**Pull request merged**